standup_notes/__main__.py

```diff
@@ -24,9 +25,11 @@ def main():
     parser.add_argument('-l', '--list', help='List all stand-up notes.', action='store_true')
     parser.add_argument('-r', '--read', help='Read stand-up notes',
                         action='store_true')
-    parser.add_argument('-c', '--copy', help='Copy stand-up notes', action='store_true')
+    parser.add_argument('-c', '--copy', help='Flag passed when using edit to copy yesterdays notes or call used to '
+                                             'copy notes to clipboard', action='store_true')
     parser.add_argument('-e', '--edit', help='Edit stand-up notes', action='store_true')
     parser.add_argument('-d', '--delete', help='Delete stand-up notes from inputted date', action='store', type=str)
+    parser.add_argument('-p', '--post', help="Post specified days chats to mattermost", action = 'store_true')
```

Comment:
I guess you added a bunch of code for mattermost integration in here. Can you take it out and add it to a branch for just the mattermost integration feature?

Reply:
Will do